src/styles/blocks/header.scss

```diff
@@ -0,0 +1,63 @@
+.header {
+  font-family: Roboto, Helvetica, sans-serif;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  background-color: rgb(255,255,255);
+
+  &__logo {
+    height: 40px;
+    width: 40px;
+    display: block;
+  }
+}
+
+.nav-list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+
+  & :not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__item {
+
+    &-link {
+      text-decoration: none;
```

Comment:
you can actually reset that, it's rarely the case to set `text-decoration` to anything other than `none`